src/lib/components/Popover.svelte

```diff
@@ -88,6 +90,8 @@
     // limited by `100vw - right padding`
     max-width: calc(100vw - var(--padding));
 
+    max-height: calc(100vh - var(--popover-top) - calc(6 * var(--padding)));
```

Comment:
Have you test on iOS?

Reply:
I don't why, my local iOS simulator is not starting with Safari... so I just turned on the responsive mode in Safari:


https://github.com/user-attachments/assets/231e6300-ae87-4470-b30d-cbe5bb10e6f1

Reply:
It seems to work partially. See screenshot. `100vh` is always tricky on iOS.

<img width="534" alt="Capture d’écran 2024-11-01 à 07 11 15" src="https://github.com/user-attachments/assets/587c0096-665e-4871-a828-e4b4215cb3f0">

Reply:
I guess it's related the url bar being at the bottom and not accounted with `vh`.

For iOS maybe `svh` instead of `vh`? or `dvh`?

<img width="534" alt="Capture d’écran 2024-11-01 à 07 14 11" src="https://github.com/user-attachments/assets/1608be7f-9f64-4ff4-98e8-462c8d5bf078">

Reply:
I went with `dvh`, WDYT?


Uploading Screen Recording 2024-11-01 at 13.59.27.mov…

Reply:
Nice!

Reply:
Cool, did not knew that worked 👍

You maybe don't need the second if, like vh just default? Also safer for those old phone which don't support `@supports`?

```
@supports not (height: 100dvh) {
```

Reply:
makes sense! i removed it and make the calculation fallback to `100vh`